code/modules/xenoarchaeology/traits/xenoartifact_minors.dm

```diff
@@ -0,0 +1,430 @@
+	death = FALSE
+	name = "Sentient Xenoartifact"
+	short_desc = "You're a maleviolent sentience, possesing an ancient alien artifact."
+	flavour_text = "Return to your master..."
+	use_cooldown = TRUE
+	invisibility = 101
+	var/obj/item/xenoartifact/artifact
+
+/obj/effect/mob_spawn/sentient_artifact/Initialize(mapload, var/obj/item/xenoartifact/Z)
+	if(!Z)
+		qdel(src)
+		return FALSE
+	artifact = Z
+	return ..()
+
+/obj/effect/mob_spawn/sentient_artifact/create(ckey, name)
+	var/datum/xenoartifact_trait/minor/sentient/S = artifact.get_trait(/datum/xenoartifact_trait/minor/sentient)
+	S.setup_sentience(artifact, ckey)
+
+//============
+// Delicate, makes the artifact have limited uses
+//============
+/datum/xenoartifact_trait/minor/delicate
+	desc = "Fragile"
+	label_desc = "Fragile: The Artifact is poorly made. Continuous use will destroy it."
+	flags = BLUESPACE_TRAIT | PLASMA_TRAIT | URANIUM_TRAIT
+
+/datum/xenoartifact_trait/minor/delicate/on_init(obj/item/xenoartifact/X)
+	X.max_integrity = pick(200, 300, 500, 800, 1000)
+	X.obj_integrity = X.max_integrity
+	X.alpha = X.alpha * 0.55
+
+/datum/xenoartifact_trait/minor/delicate/activate(obj/item/xenoartifact/X, atom/user)
+	if(X.obj_integrity)
+		X.obj_integrity -= 100
+	else if(X.obj_integrity <= 0)
+		X.visible_message("<span class='danger'>The [X.name] shatters!</span>", "<span class='danger'>The [X.name] shatters!</span>")
+		var/obj/effect/decal/cleanable/ash/A = new(get_turf(X))
+		A.color = X.material
+		playsound(get_turf(X), 'sound/effects/glassbr1.ogg', 50, TRUE) 
+		qdel(X)
+
+//============
+// Aura, adds everything in the vicinity to the target list
+//============
+/datum/xenoartifact_trait/minor/aura
+	desc = "Expansive"
+	label_desc = "Expansive: The Artifact's surface reaches towards every creature in the room. Even the empty space behind you..."
+	blacklist_traits = list(/datum/xenoartifact_trait/major/timestop, /datum/xenoartifact_trait/minor/long)
+	flags = BLUESPACE_TRAIT | PLASMA_TRAIT | URANIUM_TRAIT
+
+/datum/xenoartifact_trait/minor/aura/on_init(obj/item/xenoartifact/X)
+	X.max_range += 2
+
+/datum/xenoartifact_trait/minor/aura/activate(obj/item/xenoartifact/X)
+	for(var/atom/M in oview(min(X.max_range, 5), get_turf(X.loc)))
+		if(X.true_target.len >= XENOA_MAX_TARGETS)
+			return
+		var/obj/item/I = M
+		if(istype(M, /mob/living))
+			X.true_target |= X.process_target(M)
+		else if(istype(I) && !(I.anchored))
+			X.true_target |= X.process_target(I)
+
+//============
+// Long, makes the artifact ranged, allows effects to select targets from afar
+//============
+/datum/xenoartifact_trait/minor/long
+	desc = "Scoped"
+	label_desc = "Scoped: The Artifact has an almost magnifying effect to it. You could probably target someone from really far away with it."
+	blacklist_traits = list(/datum/xenoartifact_trait/minor/aura)
+	flags = BLUESPACE_TRAIT | PLASMA_TRAIT | URANIUM_TRAIT
+
+/datum/xenoartifact_trait/minor/long/on_init(obj/item/xenoartifact/X)
+	X.max_range += 18
```

Comment:
Combined with the one above, these basically do the same thing and are very hard to detect in game